typescript/react/portability/i18next/useselect.yaml

```diff
@@ -0,0 +1,18 @@
+rules:
+- id: react-useselect-label-not-internationalized
+  patterns:
+  - pattern: useSelect($X1, $X2, '$LABEL', $X4)
+  - pattern-not: useSelect($X1, $X2, t('...'), $X4)
+  message: React useSelect() label is not internationalized - '$LABEL'
+  languages:
+  - ts
+  - js
+  severity: WARNING
+  metadata:
+    category: portability
+    technology:
+    - react
+    source-rule-url: https://www.soluvas.com/
+    references:
+    - https://www.notion.so/hendyirawan/Internationalization-Localization-Policy-318c21674e5f44c48d6f136a6eb2e024
```

Comment:
Heya! Any chance you could make these notion pages public? 🙏

Reply:
@underyx I don't have access to this repo, so can you change it for me:

https://hendyirawan.notion.site/Internationalization-Localization-Policy-318c21674e5f44c48d6f136a6eb2e024